packages/synapse-react-client/src/components/SynapseSearchPageResults/SynapseSearchResultsCard.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Box,
+  Paper,
+  PaperProps,
+  Button,
+  styled,
+  Typography,
+  Chip,
+} from '@mui/material'
+import { StyledComponent } from '@emotion/styled'
+import FavoriteButton from '../favorites/FavoriteButton'
+import {
+  Update as UpdateIcon,
+  Download as DownloadIcon,
+} from '@mui/icons-material'
+
+export type SynapseSearchResultsCardProps = {
+  name: string
+  entityType: string
+  modifiedOn: Date
+}
+
+const SynapseSearchResultsCardContainer: StyledComponent<PaperProps> = styled(
+  Paper,
+  {
+    label: 'SynapseSearchResultsCardContainer',
+  },
+)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  minHeight: '250px',
+  width: '1000px',
+  borderRadius: '10px',
+  padding: '32px',
+  gap: '15px',
+}))
+
+export function SynapseSearchResultsCard(props: SynapseSearchResultsCardProps) {
+  return (
+    <SynapseSearchResultsCardContainer>
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          width: '100%',
+        }}
+      >
+        <Typography
+          sx={{
+            fontSize: '18px',
+            color: '#395979',
+            fontWeight: 700,
+            fontFamily: 'DM Sans',
+            textDecoration: 'underline',
+          }}
+        >
+          {props.name}
+        </Typography>
```

Comment:
I think you will get all the styles you need between the 'headline3' typography variant and using a Link
```suggestion
        <Typography
          variant='headline3'
        >
          <Link href={`${getEndpoint(BackendDestinationEnum.PORTAL_ENDPOINT)}Synapse:${entityId}`}
            {props.name}
          </Link>
        </Typography>
```